scripts/find_commits_touching_func.py

```diff
@@ -203,10 +190,14 @@ def sorter(i):
         h, s, d = get_commit_vitals(hit.commit)
         p = hit.path.split(os.path.realpath(os.curdir) + os.path.sep)[-1]
 
-        fmt = "{:%d} {:10} {:<%d} {:<%d}" % (HASH_LEN, SUBJ_LEN, PATH_LEN)
         if len(s) > SUBJ_LEN:
             s = s[: SUBJ_LEN - 5] + " ..."
-        print(fmt.format(h[:HASH_LEN], d.isoformat()[:10], s, p[-20:]))
+        print(
+            f"{h[:HASH_LEN]:{HASH_LEN}} "
+            f"{d.isoformat()[:10]:10} "
+            f"{s:<{SUBJ_LEN}} "
+            f"{p[-20:]:<{PATH_LEN}}"
```

Comment:
This is changing the behavior, can you test this and make sure it works as the original.

Reply:
In what way do you see the behaviour changed?